plugins/time-saver/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tduniec/backstage-plugin-time-saver",
-  "version": "0.1.8",
+  "version": "0.1.9",
```

Comment:
This we can also bump to major, but it also made me thinking. We can bump it as well.

Reply:
Hey @alesgurd and @tduniec. I'm giving you guys a hand and update these myself since I'm too looking forward to see how these changes will affect the latest BS version upgrade on my team.